lib/virt_autotest/utils.pm

```diff
@@ -160,12 +167,40 @@ sub is_kvm_host {
 #retrun 1 if libvirt 9.0- is running which monolithic libvirtd is the default service
 sub is_monolithic_libvirtd {
     record_info('WARNING', 'Libvirt package is not installed', result => 'fail') if (script_run('rpm -q libvirt-libs'));
-    unless (is_alp) {
-        return 1 if script_run('rpm -q libvirt-libs | grep -e "libs-9\.0" -e "libs-[1-8]\."') == 0;
-    }
+    #unless (is_alp) {
+    #    return 1 if script_run('rpm -q libvirt-libs | grep -e "libs-9\.0" -e "libs-[1-8]\."') == 0;
+    #}
     return 0;
 }
 
+# Restart libvirt daemon
+sub restart_libvirtd {
+    if (is_sle('<12')) {
+        assert_script_run('rclibvirtd restart', 180);
+    }
+    elsif (is_alp) {
+        my $_libvirtd_pid = script_output(q@ps -ef |grep [l]ibvirtd | gawk '{print $2;}'@);
+        my $_libvirtd_cmd = script_output("ps -o command $_libvirtd_pid | tail -1");
+        assert_script_run("kill -9 $_libvirtd_pid");
+        assert_script_run("$_libvirtd_cmd");
+    }
```

Comment:
According to doc, it seems that it needs to restart libvrit deamons inside container ?
See https://susedoc.github.io/doc-modular/main/single-html/alp-dolomite/#reference-kvm-server-manage

Reply:
> According to doc, it seems that it needs to restart libvrit deamons inside container ? See https://susedoc.github.io/doc-modular/main/single-html/alp-dolomite/#reference-kvm-server-manage

Good question, suppose that it was our official ALP Dolomite with modular libvirt daemons. 
Back to this PR, it just is adapted modular libvirt daemons in our SLE (i.e SLES15SP6).

Next, we'd draft new pr to adapt modular libvirt daemons in ALP Dolimite.